docs/learn/learn-comparison-ethereum-2.md

```diff
@@ -7,94 +7,92 @@ keywords: [ethereum, Ethereum, proof of stake, sharding]
-direction of Ethereum.
+The term "shards" or "sharded protocol" may be used throughout the various ecosystems. They are
+typically used, in the context of blockchains, to refer to sub-protocols, or as a general term to
+refer to a form of
+[**horizontal scaling**](https://en.wikipedia.org/wiki/Scalability#Horizontal_or_scale_out).
 
 :::
 
 ## High-Level Comparison
 
-At a high level, both protocols have fundamentally different goals, which are reflected by their
-architecture:
+Both protocols have fundamentally different goals, which can be summarized as:
+
+- Ethereum is a general-purpose blockchain, which coordinates the global coordination of the
+  Ethereum Virtual Machine (EVM). Ethereum is not specialized nor optimized for any particular
+  application, rather its primary focus is the Ethereum Virtual Machine for executing smart
+  contracts. Ethereum achieves scalability via **rollups**, which are secondary protocols which
+  utilize Ethereum as a settlement layer.
+
+- Polkadot is a multi-chain protocol that provides shared security for each one of its
+  **parachains**. Each parachain (which could also be called an "appchain" in this context) is
+  specialized towards a specific focus and optimized towards that goal. Parachains must abide by the
+  [Parachains Protocol](./learn-parachains-protocol.md)
 
-- Ethereum is a general-purpose blockchain for **global coordination**. Ethereum is not specialized
-  nor optimized for any particular application, rather its primary focus is the Ethereum Virtual
-  Machine for executing smart contracts.
-- Polkadot is a sharded blockchain that introduces shared security for each one of its shards, or
-  **parachains**. Each shard is usually specialized towards a specific focus and optimized towards
-  that goal. Polkadot provides shared security and consensus to these shards through the Polkadot
-  relay chain.
+Polkadot _does not_ directly run a virtual machine for smart contracts, as Polkadot's main purpose
+is to validate the protocols that operate under it.
 
-Polkadot can't and does not directly run something like a virtual machine for smart contracts.
-However, several of its parachains can (and do). Parachains on Polkadot can even run an EVM for
-executing smart contracts written in Solidity, Ethereum's native smart contract language.
+However, several parachains provide
+[smart contract functionality](https://wiki.polkadot.network/docs/build-smart-contracts#parachains).
+Parachains on Polkadot can even run an EVM for executing smart contracts written in Solidity using
+[Frontier, an Ethereum compatibility layer for Substrate](https://github.com/polkadot-evm/frontier).
 
-In the context of blockchain, "sharding" refers to the parallelization of state transition
-(transaction) execution. The way Ethereum and Polkadot deal with scalability and sharding is quite
-different.
+> As a general summary, one could also say that Polkadot coordinates protocols (and their subsequent
```

Comment:
I really like this! Maybe we expand a little and then finish with that sentence. We are not mentioning WASM. We could say that Ethereum is just all EVM (even if it is sharded) while Polkadot shards can have different logic as long as they compile to WASM. Does this make sense?